DOCKER_README.md

```diff
@@ -1,28 +1,30 @@
 # OpenEMR Docker Documentation
 
 ## Overview
-The OpenEMR community loves Docker. We eat and breath Docker. The OpenEMR dockers can be found on
+The OpenEMR community loves Docker. We eat and breathe Docker. The OpenEMR dockers can be found on
 [dockerhub](https://hub.docker.com/r/openemr/openemr/). There are two main categories of dockers for
 OpenEMR, Production Dockers and Development Dockers. Production dockers are meant for production use
-with tags such as `6.0.0` and `latest`. Development dockers are meant for development and include the
+with tags such as `5.0.2` and `latest`. Development dockers are meant for development and include the
```

Comment:
look like this PR is reverting recent changes (prepping for 6.0.0 to be a release)